### DISCUSSION TITLE
[TS] assign prop to observer component

### DISCUSSION THREAD

**Discussion by long76** [mobxjs/mobx#3624] [Q&A] [answered]
for example i have two components `Table` and `Cell`
why i can't set
```ts
Table.Cell = Cell;
```
i got error `Property 'Cell' not exist in type {displayName: string}`

**Comment by kubk** [comment]
It's seems like your question isn't related to Mobx. Try asking React/TypeScript forums

**Comment by long76** [comment]
> It's seems like your question isn't related to Mobx. Try asking React/TypeScript forums

if i not wrap functional component to `observer` all works

**Comment by long76** [comment, marked as answer]
i found solution wrap in `observer` after assign
